lua/astrocommunity/pack/ruby/init.lua

```diff
@@ -0,0 +1,29 @@
+local utils = require "astronvim.utils"
+
+return {
+  {
+    "nvim-treesitter/nvim-treesitter",
+    opts = function(_, opts)
+      if opts.ensure_installed ~= "all" then
+        opts.ensure_installed = utils.list_insert_unique(opts.ensure_installed, { "ruby" })
+      end
+    end,
+  },
+  {
+    "williamboman/mason-lspconfig.nvim",
+    opts = function(_, opts) opts.ensure_installed = utils.list_insert_unique(opts.ensure_installed, { "solargraph" }) end,
+  },
+  {
+    "jay-babu/mason-null-ls.nvim",
+    opts = function(_, opts) opts.ensure_installed = utils.list_insert_unique(opts.ensure_installed, "standardrb") end,
+  },
+  {
+    "mfussenegger/nvim-dap",
+    dependencies = {
+      -- "suketa/nvim-dap-ruby",
+      "takkii/neoruby-debugger",
+      -- config = function() require("dap-ruby").setup() end,
+      config = function() require("neoruby-debugger").setup() end,
```

Comment:
```suggestion
      "suketa/nvim-dap-ruby",
      config = function() require("dap-ruby").setup() end,
```
Not sure if we should have neoruby in the pack when most of our users wouldn't be able to use their docs.

Reply:
It doesn't really need docs.  Makes nvim-dap just work for ruby.

Reply:
It does. For example the docs mention that Ruby 3.2.x apparently doesn't build. I'd vote against it.

Reply:
It'll be too tricky to maintain a pack which has non-english docs, if we were to get questions about it, just like we do with other plugins. And the plugin itself doesn't seem to be that used(low amount of stars, and few people watching it.

Reply:
@dsisnero  friendly reminder